Sources/Request.swift

```diff
@@ -4,14 +4,17 @@ public struct Request: Message {
     public var version: Version
     public var headers: Headers
     public var body: Body
-    public var storage: [String: Any] = [:]
+    public var cookies: [String: String]
```

Comment:
> The above middleware would work in a Vapor project, because Vapor adds an extension to Request for accessing cookies from the headers, cookies: [String: String]. Anyone who sees this middleware might think that they can drag and drop it into any other S4 compatible project since it only uses S4.Middleware and S4.Request. But, if they drop it into a project that doesn't have the exact same Request extensions that Vapor has, it will not compile.

For this middleware to compile it needs to depend on `Vapor` to use Vapor's `cookies` property. So it would have to depend on `S4` and on `Vapor`.